modules/sw360/src/main/java/org/eclipse/sw360/antenna/sw360/rest/resource/releases/SW360Release.java

```diff
@@ -129,19 +136,27 @@ public void setDownloadurl(String downloadurl) {
         this.downloadurl = downloadurl;
     }
 
-
     @JsonIgnore
-    public Map<String, String> getCoordinates() {
+    public Map<String, String> getPurls() {
         return externalIds.entrySet().stream()
-                .filter(e -> SW360CoordinateKeysToArtifactCoordinates.getValues().contains(e.getKey()))
+                .filter(e -> PURL_TYPE.contains(e.getKey()))
                 .collect(Collectors.toMap(Map.Entry::getKey, Map.Entry::getValue));
     }
 
-    public SW360Release setCoordinates(Map<String, String> coordinates) {
-        externalIds.putAll(coordinates);
+    public SW360Release setPurls(Map<String, String> purls) {
+        dropAllPurls();
+
+        externalIds.putAll(purls);
+
         return this;
     }
 
+    private void dropAllPurls() {
```

Comment:
this drops the `docker` purl or other unsupported purls, without replacing it later? that should not happen.

Reply:
I think PURLs, which we do not handle, should be preserved in the release.

Reply:
Just for clarification: you suggest to overwrite the `externalIds` entries with the entries we get when `setPurls` is called? (which would happen automatically due to it being a map when using `putAll`)

Reply:
I am unsure, maybe one wants do drop the `p2` coordinates (and other supported types), if
- they are explicitly not provided or
- if they should be contained in the antenna artifact, if they were present previously.

Reply:
maybe best solution is: just remove `dropAllPurls`

Reply:
Good, point @maxhbr 
Actually, the question is, what setPurls does, Is it resetting the object or simply adding the purls given. Currently it completely resets the object concerning purls. Could be the right thing to do. How often will the case happen, that there are already purls inside the object and we add more?

When I read this, I was also unsure, if the deletion should be done, but after rethinking it now, I would propose to not delete the purls and check the concrete use cases first, before we lose data.

Reply:
Currently `setPurls()` is only called in Antenna when creating a new release from artifact data. 
Hence, currently there would never be any purls already safed in the release. The `mergeWith` function of the release merges all externalIds overall.
Hence, currently the `dropAllPurls` function is redundant.